bin/run-lint

```diff
@@ -1,21 +1,42 @@
 #!/usr/bin/env node
-const child_process = require( 'child_process' );
+
 const path = require( 'path' );
-var args = [ '--cache', '--quiet', '--ext=.js',  '--ext=.jsx' ];
+const child_process = require( 'child_process' );
+const stylish = require( 'eslint/lib/formatters/stylish' );
+const exitCodeFromEslint = require( './eslint-formatters-calypso/src/lib/exit-code' );
+
+var args = [ '--cache', '--quiet', '--ext=.js', '--ext=.jsx' ];
+
+const argsBranchName = [ 'rev-parse', '--abbrev-ref', 'HEAD' ];
+const branchName = child_process.spawnSync( 'git', argsBranchName ).stdout.toString().trim();
+
+if ( branchName === 'master' ) {
```

Comment:
Is there a chance we can move this logic out of the `run-lint` script and somehow into the `eslines` code? I would like to keep running `eslint` simple, so that people can easily do it outside of Calypso. If we always need those `format` arguments, it will be much harder to integrate with editors.